### PR TITLE
Update: `chrdavis.smartrename` version 1.1.2

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

### CODE REVIEW

**Comment by palenshus** [comment]
Validation runs any exes installed by the package, and validates that at least one of them succeeds. In this case, there are no exes in the package (only dlls), so this check fails @ChuckFerring / @denelon / @jamespik

**Comment by vedantmgoyal2009** [comment]
@msftbot merge

**Comment by vedantmgoyal2009** [comment]
Why it is not merging automatically?

**Comment by OfficialEsco** [comment]
https://github.com/microsoft/winget-pkgs/pull/20343#pullrequestreview-704858520

**Comment by vedantmgoyal2009** [comment]
Ok

File: manifests/c/chrdavis/smartrename/1.1.2/chrdavis.smartrename.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+PackageIdentifier: chrdavis.smartrename
+PackageVersion: 1.1.2
+PackageLocale: en-US
+Publisher: Chris Davis
+PublisherUrl: https://chrdavis.github.io/Smart-Rename/
+PublisherSupportUrl: https://github.com/chrdavis/SmartRename/issues
+#PrivacyUrl: 
+Author: Chris Davis
+PackageName: Smart Rename
```
Comment (ItzLevvie) [inline manifests/c/chrdavis/smartrename/1.1.2/chrdavis.smartrename.locale.en-US.yaml:12]:
```suggestion
PackageName: SmartRename
```

File: manifests/c/chrdavis/smartrename/1.1.2/chrdavis.smartrename.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: chrdavis.smartrename
+PackageVersion: 1.1.2
```
Comment (ItzLevvie) [inline manifests/c/chrdavis/smartrename/1.1.2/chrdavis.smartrename.installer.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/c/chrdavis/smartrename/1.1.2/chrdavis.smartrename.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: chrdavis.smartrename
+PackageVersion: 1.1.2
```
Comment (ItzLevvie) [inline manifests/c/chrdavis/smartrename/1.1.2/chrdavis.smartrename.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/c/chrdavis/smartrename/1.1.2/chrdavis.smartrename.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: chrdavis.smartrename
+PackageVersion: 1.1.2
```
Comment (ItzLevvie) [inline manifests/c/chrdavis/smartrename/1.1.2/chrdavis.smartrename.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/c/chrdavis/smartrename/1.1.2/chrdavis.smartrename.installer.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/c/chrdavis/smartrename/1.1.2/chrdavis.smartrename.installer.yaml:5]: **You'll have to update the version folder as well.**

File: manifests/c/chrdavis/smartrename/1.1.2/chrdavis.smartrename.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/c/chrdavis/smartrename/1.1.2/chrdavis.smartrename.yaml:5]: **You'll have to update the version folder as well.**

File: manifests/c/chrdavis/smartrename/1.1.2/chrdavis.smartrename.locale.en-US.yaml (hunk 4 above)
Comment (ItzLevvie) [reply manifests/c/chrdavis/smartrename/1.1.2/chrdavis.smartrename.locale.en-US.yaml:5]: **You'll have to update the version folder as well.**

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by vedantmgoyal2009** [comment]
Please merge this PR.

**Comment by vedantmgoyal2009** [comment]
Please merge this PR.

**Review by OfficialEsco — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**